src/components/ui/LoginForm.tsx

```diff
@@ -28,12 +30,29 @@ export const LoginForm = () => {
   }, [checkSession]);
 
   if (isLoading) {
-    return <div>Loading...</div>;
+    return (
```

Comment:
Until we implement the Loading state UI, it seems better to me to just show `Loading ...` text, rather than show the disabled button. When it displayed the same `Log in with Google` button after I successfully logged in, it felt like I failed to login or something.

@nick-y-ito @ShoeheyOt What do you guys think?

Reply:
@kanta1207 
I think it is good idea to have another message to make a user feel relieved, longer it takes, the more he/she get anxious.
On the other hands, however, if the loading time is very short(e.g. less than a sec), it might bother him/her because the message `Loading...`  displays very short time, like flashing it. 
So, how about `setTimeOut`  or something to display `Loading...` message only if the loading takes long.

Reply:
@kanta1207 
I am sorry I cannot recreate the situation you are talking about.

Reply:
At least in my environment, the login page including`Log in with Google` button is displayed  for seconds after I processed the login, and I felt it's better to just show `isLoading...` text for now, until we implement actual loading state UI.

https://github.com/nishiki-tech/nishiki-frontend/assets/99339182/580780d3-ebf8-4f3b-a20a-af3bb03cf892

Reply:
I have requested the designer to create a loading design. We could expect to receive it soon!

Reply:
After changing the the UI to loading state, using current implementation, when we reload the page, "reloading" is shown. To fix this, we need some investigation. How about dividing this task itself as a different scope?

https://github.com/nishiki-tech/nishiki-frontend/assets/25422441/027b9627-e21f-44ac-98a7-cf048e4c1fd8

Reply:
@kotaaaa 
You can divide tasks into smaller parts as per your comfort level :)

Reply:
I utilized `<Authenticator socialProviders={['google']} hideSignUp={true} />;` so far.